apps/server/src/core/logger/logger.module.ts

```diff
@@ -1,7 +1,43 @@
 import { Module } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { utilities, WinstonModule } from 'nest-winston';
+import winston from 'winston';
+import { ILoggerConfig } from './interfaces';
 import { Logger } from './logger.service';
 
+const availableLevels = {
+	emerg: 0,
+	alert: 1,
+	crit: 2,
+	error: 3,
+	http: 4,
```

Comment:
after discussing in the architecture chapter, we decided to remove the http log level, since we want to adhere to the standard syslog levels https://en.wikipedia.org/wiki/Syslog.

The messages currently logged to http should now be logged into notice, and the loglevel for nest be set to notice so we can see them.

This also means that we indeed need to keep the log levels of nest and feathers seperate for now.

Reply:
✔️